oasis/access-control.md

```diff
@@ -1,39 +1,48 @@
-title: Access control in ArangoDB Oasis
+title: Security and access control in ArangoDB Oasis
 ---
-# Access control in ArangoDB Oasis
+# Security and access control in ArangoDB Oasis
 
-Oasis has a structured set of resources that are subject to
+Oasis has a structured set of resources that are subject to security and
 access control:
 
 - Organizations
 - Projects
 - Deployments
 
 For each of these resources you can perform various operations.
-For example, you can *create* a project in an organization. This requires the
-respective **permission**, which can be granted to organization members via a
-**role** such as the pre-defined *Project Administrator*.
+For example, you can create a project in an organization and create a deployment inside a project.
 
-The association of a member with a role is called a **role binding**.
-All role bindings of a resource comprise a **policy**.
+## Locked resources
+
+In ArangoDB Oasis, you can lock the resources and some of their properties to prevent accidental deletion. When an object is locked,
+it cannot be deleted and must be unlocked first.
 
-Roles can be bound on an organization, project and deployment level (listed in the high to low level order). The lower
-levels *inherit* permissions from their parents.
+The hierarchial structure of the resources (organization-project-deployment) is used in the locking functionality: if a child resource is locked
+(for example, a project), you cannot delete the parent organization without unlocking that project first.
 
+{% hint 'info' %}
+If you lock a backup policy of a deployment or an IP allowlist, CA certificate, and IAM provider of a project, it is still possible to delete
+the corresponding parent resource without unlocking those properties first.
+{% endhint %}
+
+ 
 ## Policy
 
-To give a user (or a group of users) access to resources of Oasis, you assign
-a role to that user (or group). This is done in a **policy**.
+Various actions in ArangoDB Oasis require different permissions, which can be granted to users via
+**roles**.
+
+The association of a member with a role is called a **role binding**.
+All role bindings of a resource comprise a **policy**.
 
-A policy is a set of bindings of roles to users or groups for a specific
-resource. This means that there is a unique policy per resource (an organization, a project or a deployment).
+Roles can be bound on an organization, project and deployment level (listed in the high to low level order, with lower
+levels inheriting permissions from their parents). This means that there is a unique policy per resource (an organization, a project or a deployment).
 
 For example, an organization has exactly one policy,
 which binds roles to members of the organization. These bindings are used to
-give the users permissions to perform operations on this organization.
+give the users permissions to perform operations in this organization.
```

Comment:
Not sure this change is correct.

Technically the permissions give access to operations ON the organization, but also to operations on resources in the organization.

Reply:
"to operations on resources in the organization" - yes, exactly. But if you have a hard opinion and want to use "on", I can change it back.